### PR TITLE
Configurations

### PR DESCRIPTION
**Pull request by andreafioraldi**

### CODE REVIEW

File: libafl/src/bolts/llmp.rs [hunk 1]
```diff
@@ -475,7 +475,7 @@ unsafe fn _llmp_next_msg_ptr(last_msg: *const LlmpMsg) -> *mut LlmpMsg {
 /// May be used to restore the map by id.
 #[derive(Copy, Clone, Debug, Serialize, Deserialize)]
 pub struct LlmpDescription {
-    /// Info about the SharedMap in use
+    /// Info about the ShredMap in use
```
Comment (domenukk) [inline libafl/src/bolts/llmp.rs:478]: 🤔😁

**Comment by andreafioraldi** [comment]
@domenukk existential question: the configuration string should be in the event manager (like now) or in Fuzzer? In the last case we can just impl Named for Fuzzer and the configuration is the name of the fuzzer.

**Comment by domenukk** [comment]
`Named` for fuzzer may lead to users giving each node a different name (even though it's the same configuration). However it seems to be the cleaner way, right? Couldn't you potentially even reuse an event manager for multiple fuzzers, that way?

**Comment by andreafioraldi** [comment]
> `Named` for fuzzer may lead to users giving each node a different name (even though it's the same configuration). However it seems to be the cleaner way, right? Couldn't you potentially even reuse an event manager for multiple fuzzers, that way?

Yep true having it in Fuzzer can be confusing. So ATM I will keep in EventManager but we can always change later.

**Pull request merged**